src/OpenApi/test/Services/OpenApiDocumentService/OpenApiDocumentServiceTests.Parameters.cs

```diff
@@ -116,6 +116,34 @@ public async Task GetOpenApiParameters_SetsRequirednessForHeaderParameters()
     }
 #nullable restore
 
+#nullable disable
```

Comment:
Can you please add a comment explaining that this is required for the repro?

Reply:
I figured the test method name might be helpful but I can add the comment as well.